pkg/omf/functions/bundle/omf.bundle.add.fish

```diff
@@ -1,5 +1,10 @@
 function omf.bundle.add -a type name_or_url
-  set -l bundle $OMF_CONFIG/bundle
```

Comment:
By removing the `-l` flag, you allow `$bundle` to clutter the user's environment.
I suggest keeping this line as default, and then you can change it inside the first `if` case, only if it's a symlink. The second case is then irrelevant.